src/unix/solarish/illumos.rs

```diff
@@ -42,6 +42,9 @@ pub const F_OFD_SETLKW: ::c_int = 49;
 pub const F_FLOCK: ::c_int = 53;
 pub const F_FLOCKW: ::c_int = 54;
 
+// fcntl F_DUPFD_CLOEXEC has different values on Solaris and Illumos
```

Comment:
```suggestion
```

I would remove the comments.  The separate file structure already denotes different constants for illumos and Solaris, and the file would be full of comments like this if we noted every difference.